linux_os/guide/system/accounts/accounts-session/accounts_tmout/oval/shared.xml

```diff
@@ -1,10 +1,16 @@
 <def-group>
   <definition class="compliance" id="accounts_tmout" version="3">
     {{{ oval_metadata("Checks interactive shell timeout") }}}
+    {{% if product == "sle12" %}}
```

Comment:
I would rather keep the original criteria. I don't see any problem with the rule covering both `/etc/profile` and /etc/profile.d/*.sh` files. Maybe someone in the future would like to configure the module using `/etc/profile`, that way it would be already covered. Are you strongly opposed to keeping the original implementation in this part?

Reply:
I am not opposed and will make that update.

Reply:
Thanks